src/pudl/output/epacems.py

```diff
@@ -4,7 +4,8 @@
 from itertools import product
 from pathlib import Path
 
-import dask.dataframe as dd
+import pandas as pd
+import polars as pl
```

Comment:
Since we were only using dask in ~3 places, all closely related to EPA CEMS, I went ahead and tried to switch them all over to polars to remove the Dask dependency so we can just have one tool for dealing with larger-than-memory dataframes.